docs/src/using-turing/autodiff.md

```diff
@@ -23,7 +23,7 @@ Turing supports intermixed automatic differentiation methods for different varia
 using Turing
 
 # Define a simple Normal model with unknown mean and variance.
-@model gdemo(x, y) = begin
+@model function gdemo(x, y)
```

Comment:
Personally, I like the current syntax. But I am probably not a pure Julian.

Reply:
Yeah, I think in this case we should lean more towards the `@model function . . .` syntax just to be more consistent with convention.